### PR TITLE
Small project clean up to make generated build work

### PR DESCRIPTION
**Pull request by miniscruff**
Kept seeing a failing build when looking at the actions tab so decided to fix it, note that it might cause conflicts or require changes to other PRs.

* Fix CI by adding --force
* Fix formatting of markdown when needed
* Move utapi design into design folder
* Structure the nav bar to include all files
* Add contributing section to readme with some nice commands
* Remove duplicate github workflow
* Add edit uri and repo URL for mkdocs integration
* https://scality.github.io/citadel/ works now

### CODE REVIEW

File: docs/release/disconnecting.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ For example, Arsenal, CloudServer, and Backbeat.
 Let's treat them as we would any open-source component we use in a project.
 
 Each component must contain:
+
```
Comment (miniscruff) [inline docs/release/disconnecting.md:14]: @tcarmet this is required for the bullet point lists to appear in some forms of markdown, one of the side effects of markdown not being as defined as other formats.

File: docs/release/index.md [hunk 2]
```diff
@@ -2,12 +2,10 @@
...
+We expect this release process will:
 
-We expect this release process will: 
 * Give us a simpler, more conventional release workflow.
 * Speed our development, improving code-complete-to-GA time and end-to-end test feedback.
 * Enable us to release more and smaller.
 * Untangle our projects and enable us to manage them independently.
-* Reduce the stress around releases. 
-
-## [Disconnecting releases](./disconnecting.md)
```
Comment (miniscruff) [inline docs/release/index.md:13]: Removed this link as it is now part of the side bar.

**Review by alexanderchan-scality — APPROVED** [review]

**Review by rahulreddy — COMMENTED** [review]
Thank you for making this work 💯

File: docs/release/disconnecting.md (hunk 1 above)
Comment (tcarmet) [reply docs/release/disconnecting.md:14]: I thought it was showing as a bullet point already. Good to know, thanks :sweat_smile:

**Review by tcarmet — APPROVED** [review]
LGTM

**Pull request merged**